pydis_site/apps/api/serializers.py

```diff
@@ -338,6 +339,21 @@ def create(self, validated_data: dict) -> User:
             raise ValidationError({"id": ["User with ID already present."]})
 
 
+class NominationEntrySerializer(ModelSerializer):
+    """A class providing (de-)serialization of `NominationEntry` instances."""
+
+    nomination = PrimaryKeyRelatedField(
+        queryset=Nomination.objects.all(),
+        write_only=True
+    )
```

Comment:
Could you explain what this does? I'd expected to see a forward `PrimaryKeyRelatedField` serializer relationship in which a `Nomination` has a list with all the `NominationEntries` that belong to that specific nomination, like the `Album` model in the example in the docs having a `many=True` `PrimaryKeyRelatedField` relationship to the individual tracks on the album.

Don't we have something very similar? A single Nomination with several NominationEntries like an Album has many individual Tracks?

Ideally, I'd like to see reasoning for the choice you've made in your commit message, but that may now be hard to add after the fact, as it would rewrite history.

Reply:
I don't want to include nomination ID in the read result, but if I don't define queryset, I get `AssertionError: Relational field must provide a queryset argument, override get_queryset, or set read_only=True.`.